modules/gapi/test/common/gapi_video_tests_common.hpp

```diff
@@ -0,0 +1,179 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+#ifndef OPENCV_GAPI_VIDEO_TESTS_COMMON_HPP
+#define OPENCV_GAPI_VIDEO_TESTS_COMMON_HPP
+
+#include "gapi_tests_common.hpp"
+#include "../../include/opencv2/gapi/imgproc.hpp"
+
+namespace
+{
+using Point2fVector = std::vector<cv::Point2f>;
+using UcharVector   = std::vector<uchar>;
+using FloatVector   = std::vector<float>;
+using MatVector     = std::vector<cv::Mat>;
+
+namespace cgv = cv::gapi::video;
+}
+
+namespace opencv_test
+{
+
+namespace
+{
+void initTrackingPointsArray(Point2fVector& points, size_t width, size_t height,
+                             size_t pointsXNum, size_t pointsYNum)
+{
+    if (pointsXNum > width || pointsYNum > height)
+    {
+        FAIL() << "Specified points number is too big";
+    }
+
+    size_t stepX = width / pointsXNum;
+    size_t stepY = height / pointsYNum;
+
+
+    points.clear();
+    points.reserve(pointsXNum * pointsYNum);
+
+    for (size_t x = stepX / 2; x < width; x += stepX)
+    {
+        for (size_t y = stepY / 2; y < height; y += stepY)
+        {
+            Point2f pt(static_cast<float>(x), static_cast<float>(y));
+            points.push_back(pt);
+        }
+    }
+}
+
+struct OptFlowLKTestParams
+{
+    const std::string& fileNamePattern;
+    int format;
+    int channels;
+    const std::tuple<size_t,size_t>& pointsNum;
+    int winSize;
+    int maxLevel;
+    const cv::TermCriteria& criteria;
+    int flags;
+    double minEigThreshold;
+    const cv::GCompileArgs& compileArgs;
+};
```

Comment:
KlocWork may be unhappy without the  default initializers.

Reply:
default initializations added; for some variables in functions' bodies, too

Reply:
`int flags = -1`;

Reply:
The `flags` variable is not reinitialized by tests as it's value is 0 for all the tests (it is not a test parameter). So it should always be equal to 0. It is just a default parameter flags=0 for `calcOpticalFlow()`.

Reply:
Then just put 0 there.

Reply:
...and please initilize the other `int`/`size_t` fields.

Reply:
done